backend/app/services/llm/jobs.py

```
@@ -360,21 +380,19 @@ def execute_job(
                     )
 
                 elif safe_output["success"]:
-                    response.response.output.content.value = safe_output["data"][
-                        "safe_text"
-                    ]
+                    response.response.output.content.value = safe_output["data"]["safe_text"]
 
                     if safe_output["data"]["rephrase_needed"] == True:
```

Ruff (0.15.0)

[error] Avoid equality comparisons to `True`; use `safe_output["data"]["rephrase_needed"]:` for truth checks

Replace with `safe_output["data"]["rephrase_needed"]`

(E712)